charts/redpanda/helpers_test.go

```diff
@@ -12,8 +12,8 @@ package redpanda_test
 import (
 	"testing"
 
-	"github.com/redpanda-data/helm-charts/charts/redpanda"
-	"github.com/redpanda-data/helm-charts/pkg/rapidutil"
+	"github.com/redpanda-data/redpanda-operator/charts/redpanda"
```

Comment:
Thinking we probably need to add `./charts` to:

https://github.com/redpanda-data/redpanda-operator/blob/26e8da771b153ca798303523d78708d90c01e358/go.work#L3-L11

Reply:
Right